turbo.json

```diff
@@ -1,17 +1,19 @@
 {
   "pipeline": {
     "build": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["^build", "db-generate"],
       "outputs": [".next/**", ".expo/**"]
     },
     "lint": {
       "outputs": []
     },
     "dev": {
+      "dependsOn": ["db-generate"],
       "cache": false
     },
+    "db-generate": {},
```

Comment:
You may also want to add `cache: false` here until you can figure out which files to cache.

Same with `db-push`, `clean` and `//#clean`

Reply:
Alright thanks :)